test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
   });
 });
 
 function exportNamespaces(...namespace) {
   let schemas = exportToSchemas(namespace);
+  expect(validateSchema(schemas));
   return schemas;
 }
 
 function importFixture(name, ext='.json') {
   return JSON.parse(fs.readFileSync(`${__dirname}/fixtures/${name}${ext}`, 'utf8'));
 }
+
+function validateSchema(schema) {
+    const v = new validator();
+    const extendedSchema = importSchema();
+    return v.validate(schema, extendedSchema);
```

Comment:
Oh man.  You are absolutely correct about the arguments to `validate`.  I guess I was just trying to figure out _why_ the validator didn't seem to be working -- and when I saw the example with `4` passed in, I assumed that was the version of json-schema.  The example threw me off because `4` is not actually JSON -- it's just a number.  But now I see what it's doing.

Still, I don't understand why the validator didn't fail when I passed in JSON that didn't conform to the schema.  Have you been able to get this to fail (just to prove it actually does something)?